src/requests/features.jl

```diff
@@ -300,11 +300,12 @@ function _binding_kind(b ,server)
     end
 end
 
-function julia_getModuleAt_request(params::TextDocumentPositionParams, server::LanguageServerInstance, conn)
-    doc = getdocument(server, URI2(params.textDocument.uri))
-    offset = get_offset(doc, params.position)
-    x = get_expr1(getcst(doc), offset)
-    return get_module_of(StaticLint.retrieve_scope(x))
+JSONRPC.parse_params(::Type{Val{Symbol("julia/getModuleAt")}}, params) = TextDocumentPositionParams(params)
+function process(r::JSONRPC.Request{Val{Symbol("julia/getModuleAt")},TextDocumentPositionParams}, server)
```

Comment:
I think that was the wrong merge conflict resolve here :) We want the `function julia_getModuleAt_request(params::TextDocumentPositionParams, server::LanguageServerInstance, conn)` version, and neither `process` nor `parse_params`.